source/MaterialXGenGlsl/Nodes/BlurNodeGlsl.h

```diff
@@ -0,0 +1,24 @@
+//
+// TM & (c) 2020 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_BLURNODEGLSL_H
+#define MATERIALX_BLURNODEGLSL_H
+
+#include <MaterialXGenShader/Nodes/BlurNode.h>
+
+namespace MaterialX
+{
+
+/// Blur node implementation for GLSL
+class BlurNodeGlsl : public BlurNode
+{
+public:
+    static ShaderNodeImplPtr create();
+    void emitSamplingFunctionDefinition(const ShaderNode& node, GenContext& context, ShaderStage& stage) const override;
```

Comment:
Specific blur node for GLSL to emit the right function definitions for sampling.